Islandora-CLAW/Crayfish-Commons
# Upgrade 5.4 symfony and pull out EntityMapper

**GitHub Issue**: https://github.com/Islandora/documentation/issues/2220

Relies on:
* https://github.com/Islandora/chullo/pull/99

# What does this Pull Request do?

* Upgrades all the microservices from Symfony 4.4 to 5.4
* Updates dependency on chullo to PR - https://github.com/Islandora/chullo/pull/99
* Removes the EntityMapper which is now in https://github.com/Islandora/islandora-fedora-entity-mapper
* Removes the custom JWT authentication code
* Adds a way to disable the Apix Middleware

# What's new?

* Does this change require documentation to be updated? Included in this PR
* Does this change add any new dependencies? no
* Does this change require any other modifications to be made to the repository (i.e. Regeneration activity, etc.)? no
* Could this change impact execution of existing code? no

# How should this be tested?

See the full build PR on islandora-playbook here (https://github.com/Islandora-Devops/islandora-playbook/pull/263)

# Interested parties
Tag (@ mention) interested parties or, if unsure, @Islandora/committers
